src/client/entity-editor/relationship-editor/actions.ts

```diff
@@ -51,6 +53,44 @@ export function addRelationship(data: Relationship): Action {
 	};
 }
 
+export function sortRelationships(oldIndex, newIndex):any {
```

Comment:
This function could do with inline comments. It's pretty hard to follow what's happening and a new contributor would need to know for example what `orderTypeValue = 1` means.
It would be good to precede each step in the process with a line or two explaining what/why we're doing it.
I don't understand all the code in this method, like why it seems we're going from arrays to objects and back